site/profile/manifests/core/x2go_agent.pp

```diff
@@ -0,0 +1,20 @@
+# @summary
+#   Adds additional packages for x2go agent
+
+class profile::core::x2go_agent {
+
+  $packages = [
+    'x2goserver',
+    'x2goclient',
+    'x2goserver-common',
+    'x2goserver-xsession',
+    'x2goagent',
+  ]
+
+  package { $packages:
```

Comment:
It is good practice to use the `ensure_packages()` function to prevent resource conflicts with other classes.

Reply:
ok